keyboards/trnthsn/tyson60s/keyboard.json

```diff
@@ -0,0 +1,400 @@
+{
+    "manufacturer": "TrnThSnMech",
+    "keyboard_name": "Tyson60s",
+    "maintainer": "TrnThSnMech",
+    "bootloader": "rp2040",
+    "build": {
+        "debounce_type": "sym_defer_pk"
+    },
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgblight": true
+    },
+    "matrix_pins": {
+        "cols": ["GP16", "GP24", "GP18", "GP19", "GP20", "GP22", "GP5", "GP6", "GP8", "GP9", "GP10", "GP11", "GP12", "GP13"],    
+        "rows": ["GP0", "GP23", "GP2", "GP4", "GP17"]
+    },
+    "processor": "RP2040",
+    "rgblight": {
+        "animations": {
+            "alternating": true,
+            "breathing": true,
+            "christmas": true,
+            "knight": true,
+            "rainbow_mood": true,
+            "rainbow_swirl": true,
+            "rgb_test": true,
+            "snake": true,
+            "static_gradient": true,
+            "twinkle": true
+        },
+        "led_count": 20,
+        "max_brightness": 240,
+        "sleep": true
+    },
+    "usb": {
+        "vid": "0x5453",
+        "pid": "0x2311",
+        "device_version": "0.0.1"
+    },
+    "ws2812": {
+        "driver": "vendor",
+        "pin": "GP29"
+    },
+    "layout_aliases": {
+        "LAYOUT_all": "LAYOUT_60_tsangan_hhkb"
+    },
+    "community_layouts": [
+        "60_ansi_tsangan",
+        "60_tsangan_hhkb",
```

Comment:
Just a heads up, `60_tsangan_hhkb` will be deprecated in favour of `60_ansi_tsangan_split_bs_rshift` as of #24186.